myscripts.js

```diff
@@ -29,3 +30,106 @@ menuLink3.addEventListener('click', () => {
   menuButton.classList.remove('isActive');
   menu.classList.remove('isActive');
 });
+
+const projectsDetails = [
+  {
+    id: 1,
+    name: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    technologies: ['CSS', 'HTML', 'RUBY', 'BOOTSTRAP'],
+    image: './images/number1.jpg',
+    link: 'https://github.com/ReAliens/Portofilo',
+  },
+  {
+    id: 2,
+    name: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    technologies: ['CSS', 'HTML', 'RUBY', 'BOOTSTRAP'],
+    image: './images/number2.jpg',
+    link: 'https://github.com/ReAliens/Portofilo',
+  },
+  {
+    id: 3,
+    name: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    technologies: ['CSS', 'HTML', 'RUBY', 'BOOTSTRAP'],
+    image: './images/number3.jpg',
+    link: 'https://github.com/ReAliens/Portofilo',
+  },
+  {
+    id: 4,
+    name: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    technologies: ['CSS', 'HTML', 'RUBY', 'BOOTSTRAP'],
+    image: './images/number4.jpg',
+    link: 'https://github.com/ReAliens/Portofilo',
+  },
+];
+
+const addElements = () => {
+  projectsDetails.map((project) => {
+    const projectSummary = `
+    <section class='work ws-${project.id}' id='third-section'>
+    <img src='${project.image}' alt='cover' />
+    <div class='work-story'>
+    <h2>${project.name}</h2>
+    <p>
+    ${project.description}
+    </p>
+    <ul class='w-section-tech'>
+    ${project.technologies
+    .map((item, index) => `<li class=${index === 3 ? 'last-tech' : ''}>${item}</li>`)
+    .join('')}
+      </ul>
+      <button data-modal=${
+  project.id
+} type='button' class='viewButton'>See Project</button>
+      </section>`;
+    return workPart.insertAdjacentHTML('beforeend', projectSummary);
+  });
+};
+
+window.onload = addElements();
```

Comment:
- [ ] Good job loading this part of the script once the window has loaded. Unfortunately, there is still a little error here. You'll see. JavaScript runs functions automatically if they're followed by `()`. Instead, we want the function to run when the page has completely loaded. There are two ways to do this is, simply removing the `()` or calling our function inside the definition of an anonymous function.

Option 1:
```
window.onload = addElements;
```
Option 2:
```
window.onload = () => {
  addElements()
};
```

I hope this is clarifying! :heart_decoration:

Reply:
thanks